spec/object/directory_spec.rb

```diff
@@ -92,6 +92,7 @@
   end
```

Comment:
If git doesn't do the right thing, you'll want to move your changes into `spec/ocfl/object/directory_spec.rb`